community/Hexo.gitignore

```diff
@@ -0,0 +1,19 @@
+# gitignore template for Hexo sites
+# website: https://hexo.io/
+# Recommended: Node.gitignore
+
+# Ignore generated files and directories
+.tmp*
+dist/
+public/
+coverage/
+
+# Ignore temp files and lock files
+tmp/
+yarn.lock
+package-lock.json
+pnpm-lock.yaml
+
+# Add any additional patterns here
+db.json
+.deploy_git/
```

Comment:
Based on the example `.gitignore` file from the hexo-starter repo you shared, it may make sense to change this to `.deploy*` [like they do](https://github.com/hexojs/hexo-starter/blob/master/.gitignore#L7). Let me know what you think!

Reply:
ok, I'll change that